src/odemis/dataio/test/hdf5_test.py

```diff
@@ -370,6 +370,8 @@ def testMetadata(self):
         """
         size = (512, 256, 1)
         dtype = numpy.dtype("uint16")
+        # list instead of tuple for binning because json only uses lists
+        extra_md = {"Camera" : {'binning' : [0, 0]}, u"¤³ß": {'</Image>': '</Image>'}}
```

Comment:
I think that the binning value should be stored as a *tuple*, and you would compute a "expected" metadata with the list instead of tuple. Either explicitly change it, or compute it by just `json.loads(json.dumps(extra_md))`.

Also, please add units, with some `None`, and some strings.